Dockerfile

```diff
@@ -128,6 +128,12 @@ RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/so
 RUN DEBIAN_FRONTEND=noninteractive apt-get update -qq && sudo apt-get install -y yarn
 
 
+# Install Pip
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
```

Comment:
Can you please add this here https://github.com/bitrise-docker/bitrise-base/blob/master/Dockerfile#L69 , maybe in a `# Python` group? Just for a bit smaller image